dotnet/src/messages/public/BasePublicEventMessage.cs

```diff
@@ -0,0 +1,33 @@
+
+namespace ProcessEngine.ConsumerAPI.Contracts
```

Comment:
```suggestion
namespace ProcessEngine.ConsumerAPI.Contracts.Messages
```

This should be a more concise namespace.

Reply:
That's actually the same namespace we use in the .TS implementation, so I'm all for this. 👍

Reply:
We should remove the `Message` suffix on the TypeScript implementation as well then. I copied the name to keep the naming consistent.

Reply:
I was referring to @cmg-dev suggestion. That's the way I'd like to see it, because that would reflect the namespace we use with .TS. :wink: